blockchain/fullblocktests/generate.go

```diff
@@ -175,10 +175,10 @@ func makeSpendableOut(block *wire.MsgBlock, txIndex, txOutIndex uint32) spendabl
 	return makeSpendableOutForTx(block.Transactions[txIndex], txOutIndex)
 }
 
-// testGenerator houses state used to easy the process of generating test blocks
+// TestGenerator houses state used to easy the process of generating test blocks
```

Comment:
Exporting of the TestGenerator should be in a separate commit.

Reply:
Done